e2e/common/cli/delete_test.go

```diff
@@ -38,20 +38,20 @@ func TestKamelCLIDelete(t *testing.T) {
 			Expect(Kamel("run", "-n", ns, "../files/yaml.yaml").Execute()).To(Succeed())
 			Eventually(IntegrationPodPhase(ns, "yaml"), TestTimeoutMedium).Should(Equal(v1.PodRunning))
 			Expect(Kamel("delete", "yaml", "-n", ns).Execute()).To(Succeed())
-			Expect(GetOutputString(Kamel("get", "-n", ns))).ShouldNot(ContainSubstring("yaml"))
+			Eventually(Integration("yaml", ns)).Should(BeNil())
```

Comment:
@vsokolov00 can you verify that the IntegrationPod was removed as well?

Reply:
done, please check if everything is ok in the latest commits